Dirs files naming convention fix

elpaxoudis opened the pull request:
I hope I got everything right.
**Solves:** #315

AnshulMalik commented:
This is great work!

I found two files which are left somehow :P

arithmetic_analysis/Bisection.py
arithmetic_analysis/Intersection.py

elpaxoudis commented:
@AnshulMalik I think I got it now. Something strange was happening, the files in the directory were lowercase and when I changed them into uppercase again there was no change in git. Hence, I changed them to something else and then back to lowercase and I guess it worked, right?
